lms/djangoapps/teams/static/teams/js/spec/views/team_profile_spec.js

```diff
@@ -38,13 +53,105 @@ define([
+                    var view = createTeamProfileView(requests, {});
+                    expect(view.$('.team-country').length).toBe(0);
+                    expect(view.$('.team-language').length).toBe(0);
+                });
+            });
+
+            describe('Member', function() {
+
+                it('can render itself', function() {
+                    var requests = AjaxHelpers.requests(this);
+                    var view = createTeamProfileView(requests, {
+                        country: 'US',
+                        language: 'en',
+                        membership: [{'user':{'username': 'bilbo'}}]
+                    });
+                    assertTeamDetails(view, 1);
+                    expect(view.$('.team-member-status').text().trim()).toBe('You are a member of this team.');
+
+                    // assert tooltip text.
+                    expect(view.$('.member-profile p').text()).toBe('bilbo');
+                    // assert user profile page url.
+                    expect(view.$('.member-profile').attr('href')).toBe('/u/bilbo');
+
+                    //Verify that invite and leave team sections are present
+                    expect(view.$('.leave-team-link').text()).toContain('Leave Team');
+                    expect(view.$('.invites-header').text()).toContain('Invite Others');
+                    expect(view.$('.invite-text').text()).toContain('Send this link to friends so that they can join too.');
+                    expect(view.$('.invite-link-input').length).toBe(1);
+
+                });
+                it('cannot see invite url box if team is full', function() {
+                    var requests = AjaxHelpers.requests(this);
+                    var view = createTeamProfileView(requests , {
+                        country: 'US',
+                        language: 'en',
+                        membership: [{'user':{'username': 'bilbo'}},{'user':{'username': 'bilbo1'}},{'user':{'username': 'bilbo2'}}]
+                    });
+
+                    assertTeamDetails(view, 3);
+                    expect(view.$('.invites-header').text()).toContain('Invite Others');
+                    expect(view.$('.invite-text').text()).toContain('No invitations are available. This team is full.');
+                    expect(view.$('.invite-link-input').length).toBe(0);
+                });
+                it('can see & select invite url if team has capacity', function() {
+                    var requests = AjaxHelpers.requests(this);
+                    spyOn(TeamProfileView.prototype, 'selectText');
+
+                    var view = createTeamProfileView(requests, {
+                        country: 'US',
+                        language: 'en',
+                        membership: [{'user':{'username': 'bilbo'}}]
+                    });
+                    assertTeamDetails(view, 1);
+
+                    expect(view.$('.invite-link-input').length).toBe(1);
```

Comment:
I'm already checking this in this `test_team_member_can_see_invite_link` bokchoy.